### DISCUSSION TITLE
Filetree Support

### DISCUSSION THREAD

**Discussion by KittyBorgX** [helix-editor/helix#6135] [Q&A] [answered]
Is there any current support for filetree in helix? Something similar to neotree? 
If not, are there any workarounds for the same or any upcoming plans for it?

**Comment by michaeljones** [comment, marked as answer]
This might be what you're looking for: https://github.com/helix-editor/helix/pull/5768 - Still a work in progress.

**Reply by KittyBorgX** [reply]
Thank you!
